shared/src/business/useCases/trialSessions/setNoticesForCalendaredTrialSessionInteractor.test.js

```diff
@@ -114,13 +124,81 @@ describe('setNoticesForCalendaredTrialSessionInteractor', () => {
     expect(pdfDoc.getPages().length).toBe(0);
   });
 
+  it('should NOT attempt to start a trial session calendering event if its already processing or completed', async () => {
+    applicationContext
+      .getPersistenceGateway()
+      .getTrialSessionProcessingStatus.mockResolvedValueOnce('processing');
+
+    await setNoticesForCalendaredTrialSessionInteractor(applicationContext, {
+      trialSessionId,
+    });
+    expect(
+      applicationContext.getPersistenceGateway()
+        .setTrialSessionProcessingStatus,
+    ).not.toHaveBeenCalled();
```

Comment:
I think the more accurate function to test is

```
applicationContext .getMessageGateway()
      .sendSetTrialSessionCalendarEvent(...)
```

This is the function that actually triggers the process of creating the notices, right?

Reply:
Addressed.